src/periodic_table.py

```diff
@@ -0,0 +1,359 @@
+           'electronegativity': 1.3, 'type': NON_METAL},
+    '100': {'atomic_num': 100, 'symbol': 'Fm', 'name': 'Fermium', 'group': None, 'period': 7, 'precise': False,
+            'atomic_weight': 257, 'density': 9.7, 'melting_point': 1125, 'boiling_point': None, 'heat_capacity': None,
+            'electronegativity': 1.3, 'type': NON_METAL},
+    '101': {'atomic_num': 101, 'symbol': 'Md', 'name': 'Mendelevium', 'group': None, 'period': 7, 'precise': False,
+            'atomic_weight': 258, 'density': 10.3, 'melting_point': 1100, 'boiling_point': None, 'heat_capacity': None,
+            'electronegativity': 1.3, 'type': NON_METAL},
+    '102': {'atomic_num': 102, 'symbol': 'No', 'name': 'Nobelium', 'group': None, 'period': 7, 'precise': False,
+            'atomic_weight': 259, 'density': 9.9, 'melting_point': 1100, 'boiling_point': None, 'heat_capacity': None,
+            'electronegativity': 1.3, 'type': NON_METAL},
+    '103': {'atomic_num': 103, 'symbol': 'Lr', 'name': 'Lawrencium', 'group': None, 'period': 7, 'precise': False,
+            'atomic_weight': 266, 'density': 15.6, 'melting_point': 1900, 'boiling_point': None, 'heat_capacity': None,
+            'electronegativity': 1.3, 'type': NON_METAL},
+    '104': {'atomic_num': 104, 'symbol': 'Rf', 'name': 'Rutherfordium', 'group': 4, 'period': 7, 'precise': False,
+            'atomic_weight': 267, 'density': 23.2, 'melting_point': 2400, 'boiling_point': 5800, 'heat_capacity': None,
+            'electronegativity': None, 'type': METAL},
+    '105': {'atomic_num': 105, 'symbol': 'Db', 'name': 'Dubnium', 'group': 5, 'period': 7, 'precise': False,
+            'atomic_weight': 268, 'density': 29.3, 'melting_point': None, 'boiling_point': None, 'heat_capacity': None,
+            'electronegativity': None, 'type': METAL},
+    '106': {'atomic_num': 106, 'symbol': 'Sg', 'name': 'Seaborgium', 'group': 6, 'period': 7, 'precise': False,
+            'atomic_weight': 269, 'density': 35.0, 'melting_point': None, 'boiling_point': None, 'heat_capacity': None,
+            'electronegativity': None, 'type': METAL},
+    '107': {'atomic_num': 107, 'symbol': 'Bh', 'name': 'Bohrium', 'group': 7, 'period': 7, 'precise': False,
+            'atomic_weight': 270, 'density': 37.1, 'melting_point': None, 'boiling_point': None, 'heat_capacity': None,
+            'electronegativity': None, 'type': METAL},
+    '108': {'atomic_num': 108, 'symbol': 'Hs', 'name': 'Hassium', 'group': 8, 'period': 7, 'precise': False,
+            'atomic_weight': 270, 'density': 40.7, 'melting_point': None, 'boiling_point': None, 'heat_capacity': None,
+            'electronegativity': None, 'type': METAL},
+    '109': {'atomic_num': 109, 'symbol': 'Mt', 'name': 'Meitnerium', 'group': 9, 'period': 7, 'precise': False,
+            'atomic_weight': 278, 'density': 37.4, 'melting_point': None, 'boiling_point': None, 'heat_capacity': None,
+            'electronegativity': None, 'type': None},
```

Comment:
For type: None these are still under research right? They are all approximations right?